### PR TITLE
Updated node v18 to fix e2e failing job

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2709,11 +2709,6 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-14.18.63.tgz#1788fa8da838dbb5f9ea994b834278205db6ca2b"
   integrity sha512-fAtCfv4jJg+ExtXhvCkCqUKZ+4ok/JQk01qDKhL5BDDoS3AxKXhV5/MAVUZyQnSEd2GT92fkgZl0pz0Q0AzcIQ==
 
-"@types/node@^16.18.39":
```
Comment (praveen-garg) [inline yarn.lock:2712]: What was the use of this?

File: e2e/po/app.skip.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ export class SkipLogin {
   }
 
   getAppStatus() {
-    cy.get('#app > app-root > div > app-navbar > nav > div.navbar-menu .icon').should('be.visible')
```
Comment (praveen-garg) [inline e2e/po/app.skip.ts:37]: should we try something like cy.wait(2000)?

**Review by praveen-garg — APPROVED** [review]

File: yarn.lock (hunk 1 above)
Comment (mshariq-nerd) [reply yarn.lock:2712]: This package is used to load in all type definitions when using typescript in node.

File: e2e/po/app.skip.ts [hunk 3]
```diff
@@ -34,7 +34,8 @@ export class SkipLogin {
...
+    cy.wait(2000); // slight delay before checking app status
+    cy.get('app-navbar .navbar-menu .icon').should('be.visible')
```
Comment (praveen-garg) [inline e2e/po/app.skip.ts:38]: copy selector gives "#hover-link div span.icon" or just ""#hover-link div span" should be tried? or we can give a special ID to this and just get that ID.

**Review by hemantgarg923 — APPROVED** [review]

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**